archive/j/javascript/longest-word.js

```diff
@@ -0,0 +1,23 @@
+const error_msg = "Usage: please provide a string";
+if (process.argv.length != 3) {
+  console.log(error_msg);
+  process.exit(1);
+}
+
+const inputString = process.argv[2].replace(/\\n|\\t|\\r/g, " ");
```

Comment:
I think this is what is causing the [test failure](https://github.com/TheRenegadeCoder/sample-programs/actions/runs/6617388512/job/17973587051?pr=3501#step:6:5070)
```suggestion
const inputString = process.argv[2].replace(/\n|\t|\r/g, " ");
```